lib/Dialect/Secret/IR/SecretPatterns.cpp

```diff
@@ -560,6 +560,10 @@ LogicalResult HoistPlaintextOps::matchAndRewrite(
     if (isa<YieldOp>(op)) {
       return false;
     }
+    // complex op
```

Comment:
thank you! a very quick suggestion for a comment here
```suggestion
    // Conservatively preserve a complex op with a nested region
    // This could be a replaced with a recursive call to check that all of the regions' operations can be hoisted.
```